proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10029,39 +10029,508 @@ function bufferResize_( test )
+  test.identical( got, expected );
+
+  test.case = 'I64x to F32x';
+  var src = new I64x( _.bigIntsFrom( [ 1, 2, 3 ] ) );
+  var got = _.bufferRetype(src, F32x);
+  var expected = new F32x( [ 1.401298464324817e-45, 0, 2.802596928649634e-45, 0, 4.203895392974451e-45, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'I64x to U64x';
+  var src = new I64x ( _.bigIntsFrom( [ 1, 2 ] ) );
+  var got = _.bufferRetype( src, U64x );
+  var expected = new U64x( [  ] );
+  test.identical( got, expected );
+
+  /* F32x */
+
+  test.case = 'empty F32x to U8x ';
+  var src = new F32x( 3 );
+  var got = _.bufferRetype(src, U8x);
+  var expected = new U8x( [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'empty F32x to U16x ';
+  var src = new F32x( 3 );
+  var got = _.bufferRetype(src, U16x);
+  var expected = new U16x( [ 0, 0, 0, 0, 0, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'empty F32x to I32x ';
+  var view1 = new F32x( 3 );
+  var got = _.bufferRetype(src, I32x);
+  var expected = new I32x( [  0, 0, 0  ] );
+  test.identical( got, expected );
+
+  test.case = 'empty F32x to I64x ';
+  var src = new F32x( 3 );
+  var got = _.bufferRetype(src, I64x);
+  var expected = new I64x( [ 0 ] );
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'F32x to U8x';
+  var src = new F32x( [ -2, 1 ] );
+  var got = _.bufferRetype(src, U8x);
+  var expected = new U8x( [ 254, 255, 255, 255, 1, 0, 0, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'F32x to U16x';
+  var src = new F32x( [ -2, 1 ] );
+  var got = _.bufferRetype(src, U16x);
+  var expected = new U16x( [ 65534, 65535, 1, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'F32x to I32x';
+  var src = new F32x ( [ -1, 1 ] );
+  var got = _.bufferRetype(src, I32x);
+  var expected = new I32x( [ ] );
+  test.identical( got, expected );
+
+  test.case = 'F32x to U64x';
+  var src = new F32x ( [ -2, 1 ] );
+  var got = _.bufferRetype( src, U64x );
+  var expected = new U64x( [  ] );
+  test.identical( got, expected );
+
+  /* BufferView */
+
+  test.case = 'empty BufferView to U8x';
+  var src = new BufferView( new BufferRaw( 5 ) );
+  var got = _.bufferRetype( src, U8x );
+  var expected = new U8x( [ 0, 0, 0, 0, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferView to I16x';
+  var src = new BufferView( new BufferRaw( 5 ) );
+  var got = _.bufferRetype( src, I16x );
+  var expected = new I16x( [ 0, 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'empty BufferView to F32x';
+  var src = new BufferView( new BufferRaw( 5 ) );
+  var got = _.bufferRetype( src, F32x );
+  var expected = new F32x( [ 0 ] );
+  test.identical( got, expected );
+
+  test.case = 'empty BufferView to U64x';
+  var src = new BufferView( new BufferRaw( 5 ) );
+  var got = _.bufferRetype( src, U64x );
+  var expected = new U64x( [ 0 ] );
+  test.identical( got, expected );
+
```

Comment:
Please, make more commits. 
For example:
finish single test routine -> make commit -> push commit
finish correction of documentation -> make commit -> push commit

It's little steps. It's separate tasks.